src/main/java/de/tum/in/www1/artemis/web/rest/admin/AdminStatisticsResource.java

```diff
@@ -0,0 +1,49 @@
+package de.tum.in.www1.artemis.web.rest.admin;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import de.tum.in.www1.artemis.domain.enumeration.GraphType;
+import de.tum.in.www1.artemis.domain.enumeration.SpanType;
+import de.tum.in.www1.artemis.domain.enumeration.StatisticsView;
+import de.tum.in.www1.artemis.security.annotations.EnforceAdmin;
+import de.tum.in.www1.artemis.service.StatisticsService;
+import de.tum.in.www1.artemis.web.rest.StatisticsResource;
+
+/**
+ * REST controller for administrating statistics.
+ */
+@RestController
+@RequestMapping("api/admin/")
+public class AdminStatisticsResource {
+
+    private final Logger log = LoggerFactory.getLogger(StatisticsResource.class);
```

Comment:
❌ Required Change: The logger seems to refer to the wrong class. Please check the other occurrences also